client-src/elements/chromedash-form-field.js

```diff
@@ -81,6 +95,7 @@ export class ChromedashFormField extends LitElement {
     } else {
       fieldValue = e.target.value;
     }
+    this.value = fieldValue; // Should we do this?
```

Comment:
Is this comment resolved now?  It seems like you are doing it.

Reply:
The comment was a reminder to myself to be careful about doing this.  We currently don't change the value property, but is there a reason why we should not do so?  It is a dynamic property, so there could potentially be an update loop, for example.  But since this is happening in the event handler for the last update, perhaps the loop is avoided?  I'm not sure.  Seems to work fine with the text fields I am checking now, but maybe other types of fields will behave differently.

Reply:
I suppose the only way is to test each field type.

Reply:
Added TODO to the comment.  I'll keep an eye on this.